app/router.js

```diff
@@ -6,7 +6,8 @@ var Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('tweet-list');
+  this.route('list', { path: '/' });
```

Comment:
index route serves same purpose as list route that you created. Please, use Index route to show the list instead of List route.